en/components/slider/slider_ticks.md

```diff
@@ -0,0 +1,255 @@
+    [showTicks]="true"
+    [primaryTicks]="11"
+    [ticksOrientation]="ticksOreintation"
+></igx-slider>
+```
+```typescript
+public ticksOrientation = TicksOrientation.Top;
+```
+
+<div class="sample-container loading" style="height: 140px">
+    <iframe id="slider-primary-ticks-top-iframe" data-src='{environment:demosBaseUrl}/interactions/slider-primary-ticks-top' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="slider-primary-ticks-top-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+We haven’t done anything special here, except that we’ve changed the **orientation** of the **ticks** and positioned them at the top of the slider. As we saw, there isn’t any **thumb label** that is popping up.
+
+#### Slider ticks with labels view
+As we said, the feature is extremely flexible and it works with the **labels view** feature as well. Let’s see how.
+
+```html
+<igx-slider
+    [labels]="labels"
+    [showTicks]="true"
+    [secondaryTicks]="3"
+></igx-slider>
+```
+```typescript
+  public type: SliderType = SliderType.RANGE;
+  public labels = ["04:00", "08:00", "12:00", "16:00", "20:00", "00:00"];
+```
+
+<div class="sample-container loading" style="height: 140px">
+    <iframe id="slider-timeframe-iframe" data-src='{environment:demosBaseUrl}/interactions/slider-timeframe' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="slider-timeframe-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+Here we do not set [`primaryTicks`]({environment:angularApiUrl}/classes/igxslidercomponent.html#primaryticks), because it won’t be reflected in any way, because the **length** of the collection takes control over it. This does not mean that we are not able to set any [`secondaryTicks`]({environment:angularApiUrl}/classes/igxslidercomponent.html#secondaryticks). We are, but they will be just empty **ticks** which would serve as some sort of indicators between the **primary ticks**.
+
+#### Template tick labels
+Lastly we will see how we can provide a custom template for our **tick labels** and what our [`context`]({environment:angularApiUrl}/classes/igxtickscomponent.html#context) provides.
+
+```html
+    <igx-slider
+    [showTicks]="true"
+    [primaryTicks]="3"
+    [secondaryTicks]="3">
+        <ng-template igxSliderTickLabel let-value let-primary="isPrimary" let-idx="index">
+            {{ tickLabel(value, primary, idx) }}
+        </ng-template>
+    </igx-slider>
+```
+Setting to the template [`IgxTickLabelTemplateDirective`]({environment:angularApiUrl}/classes/igxticklabeltemplatedirective.html) says that this is the template we need to look for when we render our **tick labels**. 
```

Comment:
I think "setting to the template" will make mroe sense, I am confused what ti means here. Maybe "Setting the template"